libs/indibase/indicontroller.cpp

```diff
@@ -97,6 +97,10 @@ bool Controller::initProperties()
     IUFillSwitchVector(&UseJoystickSP, UseJoystickS, 2, device->getDeviceName(), "USEJOYSTICK", "Joystick", OPTIONS_TAB,
                        IP_RW, ISR_1OFMANY, 0, IPS_IDLE);
 
+    IUFillText(&JoystickDeviceT[0], "SNOOP_JOYSTICK_DEVICE", "Device", "Joystick");
```

Comment:
@knro I've commented out everything locally except the additions to `initProperties` and `test_alignment` still fails on Fedora with a segfault. I haven't been able to get gdb to break on the segfault, so figuring this out is turning out to be quite difficult. Any tips or help would be appreciated.

Reply:
Not sure, nothing from your commit stands out.